src/core/layout/qgslayoutitem.cpp

```diff
@@ -95,7 +95,7 @@ QgsLayoutItem::QgsLayoutItem( QgsLayout *layout, bool manageZValue )
 
 QgsLayoutItem::~QgsLayoutItem()
 {
-  cleanup();
+  QgsLayoutItem::cleanup();
```

Comment:
this doesn't sound required?

Reply:
It silences the warning and makes it clear that only the base class method will be called